src/Viewer.tsx

```diff
@@ -852,6 +852,94 @@ function Viewer(): ReactElement {
     return [threshold.min, threshold.max];
   };
 
+  const tabItems = [
+    {
+      label: "Track plot",
+      key: TabType.TRACK_PLOT,
+      children: (
+        <div className={styles.tabContent}>
+          <PlotTab
+            setFrame={setFrameAndRender}
+            findTrackInputText={findTrackInput}
+            setFindTrackInputText={setFindTrackInput}
+            findTrack={findTrack}
+            currentFrame={currentFrame}
+            dataset={dataset}
+            featureKey={featureKey}
+            selectedTrack={selectedTrack}
+            disabled={disableUi}
+          />
+        </div>
+      ),
+    },
+    {
+      label: "Scatter plot",
+      key: TabType.SCATTER_PLOT,
+      children: (
+        <div className={styles.tabContent}>
+          <ScatterPlotTab
+            dataset={dataset}
+            currentFrame={currentFrame}
+            selectedTrack={selectedTrack}
+            findTrack={findTrack}
+            setFrame={setFrameAndRender}
+            isVisible={config.openTab === TabType.SCATTER_PLOT}
+            isPlaying={timeControls.isPlaying() || isRecording}
+            selectedFeatureKey={featureKey}
+            colorRampMin={colorRampMin}
+            colorRampMax={colorRampMax}
+            colorRamp={getColorMap(colorRampData, colorRampKey, colorRampReversed)}
+            categoricalPalette={categoricalPalette}
+            inRangeIds={inRangeLUT}
+            viewerConfig={config}
+            scatterPlotConfig={scatterPlotConfig}
+            updateScatterPlotConfig={updateScatterPlotConfig}
+            showAlert={showAlert}
+          />
```

Comment:
This is already clean and readable, I just have a thought for future maintainability if we anticipate adding more props: would it be less error prone to group related props into a single config object instead of passing them individually?

For instance: 
```
const scatterPlotDisplayConfig = {
dataset,
currentFrame,
...
};

const scatterPlotColorConfig = {
colorRampMin,
colorRampMax,
...
};

<ScatterPlotTab
  displayConfig={scatterPlotDisplayConfig}
...
>
```

Reply:
Oooh, yes! We already kind of do this for the `scatterPlotConfig` and `viewerConfig`, but you're right that the color ramps etc. could be bundled together into an object as well. I'll open an issue ticket for this and update here. :')

Reply:
Opened as https://github.com/allen-cell-animated/timelapse-colorizer/issues/502